# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/scribus/glibc234.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


A trival fix for compilation with gcc-2.3.4 ...

  - Rene Rebe <rene@rocklinux.org>

--- scribus-1.1.6/scribus/page.cpp.orig	2004-05-15 23:16:48.585516928 +0200
+++ scribus-1.1.6/scribus/page.cpp	2004-05-15 23:17:39.016850200 +0200
@@ -15,6 +15,8 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <unistd.h> // unlink
+
 #include "page.h"
 #include "page.moc"
 #include "pageitem.h"
